Gemfile

```diff
@@ -3,24 +3,16 @@ git_source(:github){|repo| "https://github.com/#{repo}.git"}
 
 ruby "2.7.1"
 
+gem "bcrypt", "3.1.13"
```

Comment:
gem này dùng làm gì em

Reply:
Đây là gem để dùng thuật toán Bcrypt mã hoá cho những thứ mình muốn mã hóa trước khi lưu vào db, ví dụ mật khẩu
note: em trả lời ở bên trên rồi ạ